persistence-dse-6.8/src/main/java/io/stargate/db/dse/impl/DsePersistence.java

```diff
@@ -311,6 +312,13 @@ public Authenticator getAuthenticator() {
                   }
 
                   final CQLStatement statement = prepared.statement;
+                  // Please note that this need to happen _before_ the beginTraceExecute, because
```

Comment:
aside: Is this a bug in the formatter? (it's done this to me too)

Reply:
Yeah, weird. I cleaned it up and the formatter seems to leave it away now.